Samples/macOS-Swift/macOS-Swift.yml

```diff
@@ -0,0 +1,133 @@
+name: macOS-Swift
+createIntermediateGroups: true
+generateEmptyDirectories: true
+configs:
+  Debug: debug
+  Test: debug
+  TestCI: debug
+  Release: release
+projectReferences:
+  Sentry:
+    path: ../../Sentry.xcodeproj
+fileGroups:
+  - ../Shared/Config
+  - macOS-Swift.yml
+options:
+  bundleIdPrefix: io.sentry
+targets:
+  macOS-Swift:
+    type: application
+    platform: auto
+    sources:
+      - Shared
+      - ../Shared/GitInjections.swift
+      - ../Shared/screenshot.png
+      - ../Shared/SampleAssets.xcassets
+    dependencies:
+      - target: Sentry/Sentry
+    configFiles:
+      Debug: macOS-Swift.xcconfig
+      Release: macOS-Swift.xcconfig
+      Test: macOS-Swift.xcconfig
+      TestCI: macOS-Swift.xcconfig
+    postCompileScripts:
+      - script: ../Shared/inject-git-info.sh
+        name: Inject Git Information into Info.plist
+        basedOnDependencyAnalysis: false
+    postBuildScripts:
+      - script: ../Shared/reset-git-info.sh
+        name: Reset Git Fields in Info.plist
+        basedOnDependencyAnalysis: false
+  macOS-Swift-Other:
+    type: application
+    platform: auto
+    sources:
+      - Shared
+      - ../Shared/GitInjections.swift
+      - ../Shared/screenshot.png
+      - ../Shared/SampleAssets.xcassets
+    dependencies:
+      - target: Sentry/Sentry
+    configFiles:
+      Debug: macOS-Swift-Other.xcconfig
+      Release: macOS-Swift-Other.xcconfig
+      Test: macOS-Swift-Other.xcconfig
+      TestCI: macOS-Swift-Other.xcconfig
+    postCompileScripts:
+      - script: ../Shared/inject-git-info.sh
+        name: Inject Git Information into Info.plist
+        basedOnDependencyAnalysis: false
+    postBuildScripts:
+      - script: ../Shared/reset-git-info.sh
+        name: Reset Git Fields in Info.plist
+        basedOnDependencyAnalysis: false
+  macOS-Swift-Sandboxed:
+    type: application
+    platform: auto
+    sources:
+      - Shared
+      - ../Shared/GitInjections.swift
+      - ../Shared/screenshot.png
+      - ../Shared/SampleAssets.xcassets
+    dependencies:
+      - target: Sentry/Sentry
+    configFiles:
+      Debug: macOS-Swift-Sandboxed.xcconfig
+      Release: macOS-Swift-Sandboxed.xcconfig
+      Test: macOS-Swift-Sandboxed.xcconfig
+      TestCI: macOS-Swift-Sandboxed.xcconfig
+    postCompileScripts:
+      - script: ../Shared/inject-git-info.sh
+        name: Inject Git Information into Info.plist
+        basedOnDependencyAnalysis: false
+    postBuildScripts:
+      - script: ../Shared/reset-git-info.sh
+        name: Reset Git Fields in Info.plist
+        basedOnDependencyAnalysis: false
+  macOS-Swift-Sandboxed-Other:
```

Comment:
`l`: This might be a good spot to add a comment that this "Other" app is used to have two isolated apps for UI tests available, to confirm that they use isolated file paths

Reply:
Indeed. One of the benefits of using a yml spec for this!